### ISSUE TITLE
Error getting phantomjs version

### ISSUE THREAD

**Issue by ablipman** [sitespeedio/sitespeed.io#657]
I installed sitespeed.io on Windows, but I'm getting this error when running it:

C:\Users\alipman\AppData\Roaming\npm>sitespeed.io.cmd -u http://www.mysite.com -b firefox
error: Error getting versions: { [Error: Command failed: C:\Users\alipman\AppData\Roaming\npm\node_modules\sitespeed.io\node_modules\phantomjs\lib\phantom\phantomjs.exe --version]
  killed: true,
  code: null,
  signal: 'SIGTERM',
  cmd: 'C:\\Users\\alipman\\AppData\\Roaming\\npm\\node_modules\\sitespeed.io\\node_modules\\phantomjs\\lib\\phantom\\phantomjs.exe --version' }
error:  Error: Command failed: C:\Users\alipman\AppData\Roaming\npm\node_modules\sitespeed.io\node_modules\phantomjs\lib\phantom\phantomjs.exe --version

    at ChildProcess.exithandler (child_process.js:751:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1015:16)
    at Process.ChildProcess._handle.onexit (child_process.js:1087:5)

When I run this:
C:\Users\alipman\AppData\Roaming\npm\node_modules\sitespeed.io\node_modules\phantomjs\lib\phantom\phantomjs.exe --version

It works fine and returns:
1.9.8

But I noticed that I don't get a cmd prompt after the version is returned.

Any ideas what's wrong?

**Comment by soulgalore** [comment]
Hi @ablipman 
hmm no. Did a quick test on my Windows image (8.1) using node 0.10.28 and there it works. What versions do you use?

Best
Oeter

**Comment by ablipman** [comment]
Thanks, Oeter.  I am using Windows 7 SP 1, node 0.12.2, and sitespeed.io
3.4.1.

Andi

**Comment by ablipman** [comment]
I figured out how to work around this issue, so I thought I would post in case anyone else has the same issue.  I realized that the phantomjs --version command was hanging, so I downloaded the latest version of phantomjs.exe (2.0.0) and copied it to C:\Users\alipman\AppData\Roaming\npm\node_modules\sitespeed.io\node_modules\phantomjs\lib\phantom, and this resolved my issue!  With this version, the --version command didn't hang.
